.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*.{js,jsx,ts,tsx,css,scss,html,md,mdx,json,yml}": "prettier --write",
+  "*.{j,t}s{,x}": "eslint --fix",
+  "src/**/*.svg": "svgo --config=.svgorc.yml"
```

Comment:
[nitpick] could probably be just `*.svg` since we don't have any other svg files, and if we did we'd want to lint them too anyways.

Reply:
That one is a little different, it is not linting but instead processing of SVGs based on configuration. That specifically is used for SVG icons, so not sure we should include any other rather than in `src`. TBH I would even be curious to restrict to `src/svg` 🤔

Reply:
Huh, I wonder if it belongs in `lint-staged` then?
It's essentially a `build` script, instead of `lint`.
Should probably just run on CI, like other `build` and `tests` scripts.

Reply:
I prefer to run optimisation once when adding, instad of each time when building...

Reply:
But you're right, maybe this should just belong as part of Husky hook.

I will merge this now, and put this on separate PR.

Reply:
Well if it modifies the file (instead of generating others) then it's technically linting.
I suppose we'd want to process/optimise all `svg`s, not only the ones in `src`?

I'd do that, but I don't mind the current or `src/svg/*.svg` either.

Reply:
Not all, e.g. this configuration strips "fill" and otgers that we don't need for icons, but other SVGs it might ruin...